test/functional/mempool_accept.py

```diff
@@ -83,7 +83,7 @@ def run_test(self):
         )
 
         self.log.info('A transaction not in the mempool')
-        fee = 0.00000700
+        fee: Decimal = 0.00000700
```

Comment:
Where is this syntax defined? It seems very ugly/weird for Python, and in my Python 3.7.8 console, it also doesn't result in a Decimal type...

Reply:
It's a type annotation. It has no operational semantics (so execution will be identical to not having them at all). All it does is fail type checking if that's enabled.

I assume OP is incorrectly assuming that it actually determines the runtime type.

Reply:
I found it here: https://docs.python.org/3/library/typing.html

Reply:
I don't think this is correct. Python never interprets inline numbers as Decimal, and the type annotation doesn't change that. I think what you want is:
```python
fee: Decimal = Decimal('0.00000700')
```

Reply:
As sipa mentioned, I assumed python has introduced something like few other languages which I missed. I should have read the details and tested few things.

@laanwj your suggestion is correct however I am not sure if we should use the type annotations if they don't change anything. Planning to make a new commit with changes like 'test4' mentioned in the below pic after trying few things on my system and doing more research:

![image](https://user-images.githubusercontent.com/13405205/94712802-95857b80-0367-11eb-8b8d-860d0123cea0.png)